src/module.cpp

```diff
@@ -3,6 +3,7 @@
 #include "standalone/hello.hpp"
 #include "standalone_async/hello_async.hpp"
 #include <nan.h>
+#include <protozero/types.hpp>
```

Comment:
What necessitated this addition?

Reply:
To be able to ensure the mason deps are actually installed (asking the compiler to confirm for us). But, we should remove now that we've tested, since of course `protozero` is not actually a dep here. Removed in ebfd02c